applications/nrf_desktop/README.rst

```diff
@@ -400,6 +400,7 @@ The following build types are available for various boards in the nRF Desktop:
 * ``b0`` -- ``debug`` build type with the support for the B0 bootloader enabled (for :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
```

Comment:
Please use consistent commit naming. You could also merge parts of this commit with commits that actually introduce given functionality (you affect 2 areas here - CAF BLE adv and nrf_desktop)